src/main/java/io/stargate/sgv2/jsonapi/service/shredding/CqlNamedValue.java

```diff
@@ -15,4 +15,15 @@ public class CqlNamedValue extends NamedValue<ColumnMetadata, Object> {
   public CqlNamedValue(ColumnMetadata name, Object value) {
     super(name, value);
   }
+
+  @Override
+  public String toString() {
+    return new StringBuilder(getClass().getSimpleName())
+        .append("{columnName=")
+        .append(name().getName())
+        .append(", value=")
+        .append(value())
```

Comment:
idea: include type (`getClass()`) of value in parenthesis? (often good to know if it's String "1" or `Long` `1` f.ex)